src/main/java/ru/yanmayak/TelegramNotifyBot/command/NotCommand.java

```diff
@@ -0,0 +1,19 @@
+package ru.yanmayak.TelegramNotifyBot.command;
+
+import org.telegram.telegrambots.meta.api.objects.Update;
+import ru.yanmayak.TelegramNotifyBot.command.serivce.SendBotMessageService;
+
+public class NotCommand implements Command {
```

Comment:
All commands are the same except message. You could use only one command class and pass its message to constructor